pay-api/migrations/versions/b624b5dbc9b9_added_pad_activation_dates.py

```diff
@@ -0,0 +1,93 @@
+"""added pad activation dates
+
+Revision ID: b624b5dbc9b9
+Revises: 7c19ee3a58aa
+Create Date: 2020-11-16 15:53:01.664968
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import String
+from sqlalchemy.sql import column, table
+
+# revision identifiers, used by Alembic.
+revision = 'b624b5dbc9b9'
+down_revision = '7c19ee3a58aa'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('cfs_account', sa.Column('pad_tos_accepted_by', sa.String(length=50), nullable=True))
```

Comment:
Better to move this to payment_account table as new cfs_account record will be created whenever anything with CFS (bank details) changes.

Reply:
done